paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -132,7 +132,18 @@ def paasta_start_or_stop(args, desired_state):
         load_deployments=False,
     )
 
-    remote_refs = remote_git.list_remote_refs(utils.get_git_url(service))
+    try:
+        remote_refs = remote_git.list_remote_refs(utils.get_git_url(service))
+    except remote_git.LSRemoteException as e:
+        msg = (
+            "Error talking to the git server: %s"
+            "This PaaSTA command requires access to the git server to operate."
+            "The git server may be down or not reachable from here."
```

Comment:
these are going to be stuck together like `... to operate.The git server ...`. Maybe stick a space at the end of each sentence.